sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/MethodParameterLookupTest.cs

```diff
@@ -36,7 +35,6 @@ namespace SonarAnalyzer.UnitTest.Helpers
     public class MethodParameterLookupTest
     {
 
-
         [TestMethod]
         public void TestMethodParameterLookup_CS()
```

Comment:
Could you please add a test, checking that `TryGetNonParamsSyntax` will correctly throw an exception when used on an actual params parameter symbol.